helix-rest/src/main/java/org/apache/helix/rest/common/HttpConstants.java

```diff
@@ -26,4 +26,6 @@
     PUT,
     DELETE
   }
+
+  public static final String HTTP_PROTOCOL = "http://";
```

Comment:
Nit: How about something like HTTP_PREFIX?

Reply:
Good point, I'll opt for `HTTP_PROTOCAL_PREFIX`. It is a protocol and it is also a prefix.